projects/angular/src/data/datagrid/datagrid-row.html

```diff
@@ -2,12 +2,17 @@
   We need to wrap the #rowContent in label element if we are in rowSelectionMode.
   Clicking of that wrapper label will equate to clicking on the whole row, which triggers the checkbox to toggle.
 -->
-<label class="datagrid-row-clickable clr-col-null" *ngIf="selection.rowSelectionMode" (mousedown)="clearRanges($event)">
+<div
+  class="datagrid-row-clickable clr-col-null"
```

Comment:
Is the `clr-col-null` class still needed? I think that was was a workaround for something that the `ClrLabel` directive does.

Reply:
Good catch. I believe it can be removed.